### DISCUSSION TITLE
下载弹窗多出一段空白区域

### DISCUSSION THREAD

**Discussion by Neo-Ze** [the1812/Bilibili-Evolved#3168] [提问 / Q&A]
粗略看了几个视频，下载弹窗均出现这种情况
浏览器：chrome 99.0.4844.82
evolved版本：2.1.7
![image](https://user-images.githubusercontent.com/81399833/159759493-164f0589-c31a-4004-9389-62d390be16e2.png)

是解决这个问题出现的吗
![image](https://user-images.githubusercontent.com/81399833/159761367-3b7623ad-ff99-42a3-b3fd-e62d4b03121a.png)

**Comment by the1812** [comment]
是的, 应该可以缩短一点
